docs/tutorials/developer-journey/level-1/1.2-motoko-lvl1.md

```diff
@@ -0,0 +1,299 @@
+
+With actors in mind, the following terms and concepts are essential to be aware of. Many of these concepts apply to several different programming languages, though we'll be focusing on their application within Motoko.
+
+- **Declaration:** A declaration in Motoko is used to define immutable variables, mutable state, objects, actors, classes, and other data types. 
+
+- **Expression:** An expression in Motoko is used to describe computations that involve declarations. 
+
+- **Program:** A program in Motoko is a collection of expressions and declarations.
+
+- **Value:** In Motoko, a value represents an entity that can be manipulated by a program. 
+
+- **Variable:** In Motoko, variables are used to store data or information that can be referenced or manipulated by a program. 
+
+- **Type:** A type in Motoko makes a precise prediction regarding the program's result. 
+
+## Motoko syntax
+
+Motoko's program syntax uses declarations and expressions. Programs consist of an actor expression that is introduced using the keyword `actor`. To introduce the Motoko syntax, first we'll use small code snippets that do not define entire programs, but rather some simple printed terminal output.
+
+For example, the following code snippet consists of two **declarations** for the variables `x` and `y`, followed by an **expression**. This forms a single **program**.
+
+```
+let x = 1; /// declaration
+let y = x + 1; /// declaration
+x * y + x; /// expression
+```
+
+In this snippet, the program's type is `Nat` (natural number), since when the program is run, the result value is a `Nat` value of 3. We dive into value types in [values](#values-and-evaluation).
+
+Now, let's build upon this small snippet. If we introduce a block with enclosing braces `do {` and `}` and another variable `z`, we can amend our original program as follows:
+
+```
+let z = do {
+  let x = 1;
+  let y = x + 1;
+  x * y + x
+};
+```
+
+The variable `z` now stores the result value of `3`, and can be called by additional methods. 
+
+Next, let's look at using and importing the base Motoko library. 
+
+## Using the base library
+
+Motoko is designed to minimize built-in types and operations, so whenever possible the Motoko base library provides the necessary operations and types for developers to utilize. 
+
+The base library includes a selection of modules that focus on the core features of Motoko. The base library is still under development, and is expected to grow in size as additional features are developed. 
+
+To import the base library, the `import` keyword can be used at the start of your Motoko code file. After the `import` keyword, you need to provide a **local module name** and a **file pat**h that the import declaration can use to locate the imported module.
+
+For example, to import a local module named 'Debug', we can use the following import statement:
+
+```
```

Comment:
Comment concerning all the code snippets. Can't we use 

```Motoko
```

for all of them? It would be nice to have a consistent color scheme.